content/blog/a-simple-guide-to-noatarizing-your-java-application/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: A simple guide to notarzing your Java application
+date: "2020-05-21T12:00:00+00:00"
+author: georgeadams
+description: Apple has recently changed the requirements for applications to install on macOS 10.15 and above.
+featuredImage: "./catalina.jpeg"
+tags:
+  - macOS
+  - Catalina
+  - Notarization
+---
+
+Apple has recently changed the requirements for applications to install on macOS 10.15 and above.
+
+![MacOS Catalina background](./catalina.jpeg)
+
+## Summary
+Apple has recently changed the requirements for applications to install on macOS 10.15 and above. The change requires developers to notarize the application before it gets shipped. Notarization involves submitting the application to Apple to be scanned and generates a JSON report with any issues. More information about notarization can be found [here](https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution).
+
+## How does this affect my runtime?
+There are a few different scenarios around today so let’s walk through them…
+
+### You’ve never code signed your runtime before:
+If you’ve always shipped your runtime without code signing it then you may not need to change anything, if a developer wishes to bundle your runtime into their application they can codesign at that stage and all will be well but…
+
+### You ship macOS installers:
+If you ship pkg or dmg installers then you are going to have to notarize the installer going forwards. In principle this means that you need to submit the installer to Apple using xcrun which comes with XCode:
```

Comment:
nit: DMGs aren't installers.
nit: Backticks around `xcrun` to format it as code? 
nit: Apple spells it `Xcode`